README.md

```diff
@@ -759,6 +759,7 @@ The OpenTelemetry Operator _might_ work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes     | Cert-Manager | Prometheus-Operator |
 |------------------------|----------------| ------------ |---------------------|
+| v0.108.0               | v1.23 to v1.30 | v1           | v0.75.0             |
```

Comment:
we support up to 1.31 now https://github.com/open-telemetry/opentelemetry-operator/pull/3246

Reply:
Updated